src/main/kotlin/lotto/domain/LottoTicket.kt

```diff
@@ -1,33 +1,33 @@
 package lotto.domain
 
-class LottoTicket(lottoNumbers: List<LottoNumber>) {
-    val value: List<LottoNumber> = lottoNumbers.sortedBy { it.value }
+class LottoTicket(val value: Set<LottoNumber>) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class LottoTicket(val value: TreeSet<LottoNumber>) {
```